java/script.js

```diff
@@ -45,4 +45,68 @@ const newarrow = document.querySelector('#closebtn');
 
 newarrow.addEventListener('click', () => {
   modalMenu.style.display = 'none';
-});
\ No newline at end of file
+});
+
+const arr1 = ['Multi-Post Stories 1', 'This is the firtst A daily for this is a test and is test number 1', 'CSS', 'Html', 'Bootstrap', 'Ruby'];
+
+const [first, second, third, fourth, fifth, sixth] = arr1;
+
+const arr2 = ['Multi-Post Stories 2', 'This is the second A daily for this is a test and is test number 2', 'CSS', 'Html', 'Bootstrap', 'Ruby'];
+
+const [one, two, three, four, five, six] = arr2;
+
+const arr3 = ['Multi-Post Stories 3', 'This is the third A daily for this is a test and is test number 3', 'CSS', 'Html', 'Bootstrap', 'Ruby'];
+
+const [name3, descrip, tec1, tec2, tec3, tec4] = arr3;
```

Comment:
- [ ] You can create an array  of objects instead so you can store the project's data and avoid repetition
e.g
```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```